src/lsg/patterns/chrome/demo.tsx

```diff
@@ -0,0 +1,30 @@
+import { Size } from '../copy';
```

Comment:
`import { Size } from '../../lsg/patterns/copy';` => `import { Size as FontSize } from '../../lsg/patterns/copy';`
With that it is more clear which size is meant. In the copy component it's kind of obvious but not here in the chrome-container

Reply:
✅